tests/raft_node.py

```diff
@@ -156,124 +167,6 @@ async def _unittest_raft_node_election_timeout() -> None:
     await asyncio.sleep(1)  # give some time for the node to close
 
 
-async def _unittest_raft_node_heartbeat() -> None:
```

Comment:
Why did you remove this test?

Reply:
I don't know why it was removed but in [this commit](https://github.com/OpenCyphal-Garage/cyraft/pull/11/commits/6073cca7fdab641e62ff235075cafbabc6d40956) I added it back

Reply:
[Fixed](https://github.com/OpenCyphal-Garage/cyraft/pull/11/commits/e9cd8165d7ba66c545d75d3037f5b044045cc01d)